<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>980</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1060</width>
    <height>980</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1060</width>
    <height>980</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblImg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edtConsole">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>824</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Generate</string>
    </property>
    <widget class="QPushButton" name="btnGen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnSiz">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="radAlgo2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Growing tree backtracer</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radAlgo3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Growing tree Prim</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLops">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>106</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Loops</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radAlgo1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Eller's</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Solve</string>
    </property>
    <widget class="QPushButton" name="btnSolv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpClrSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Color set</string>
    </property>
    <widget class="QPushButton" name="btnClrRst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Walls</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrWall">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrPath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rec path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrRecPath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bfs path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrBfsPath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrCros">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Crossover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrFadeS">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblFadeS">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fade start</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFadeE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fade end</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClrFadeE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radFadeN">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>No fade</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radFadeH">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal fade</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radFadeV">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical fade</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkClrIns">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>106</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Insanity</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
